docs/reference/index.md

```diff
@@ -1,23 +1,26 @@
 # Schema reference
 
+The Open Contracting for Infrastructure Data Standard (OC4IDS) provides a common approach for the disclosure of structured data on infrastructure projects and their related contracting processes.
+
+OC4IDS is comprised of a schema file and codelist files, and reference documentation is available for the [schema](reference.md) and [codelists](codelists.md).
+
+The schema can be explored using the [schema browser](schema.md) and can be [downloaded here](../../../../project-schema.json). The schema is expressed using [JSON Schema, draft 4](https://tools.ietf.org/html/draft-zyp-json-schema-04).
+
+OC4IDS data must be published as part of a [project package](project_package.md), which serves as a container for data on multiple projects and adds important metadata about the data publication. The [packaging data documentation](project_package.md) describes how to package OC4IDS data for publication.
```

Comment:
My earlier suggestion was unclear. I think the link in the first sentence means we no longer need the second sentence ("The [packaging data documentation](project_package.md) describes how to package OC4IDS data for publication.").

Reply:
Updated in https://github.com/open-contracting/infrastructure/pull/89/commits/c7df5e81b9bb577a9c184aeab4784b80864c3c05